UE/Application/Ports/IUserPort.hpp

```diff
@@ -10,8 +10,12 @@ class IUserEventsHandler
 public:
     virtual ~IUserEventsHandler() = default;
 
-    virtual void handleCallAccepted()=0;
-    virtual void handleCallRejected()=0;
+    virtual void USER_handleCallAccept(common::PhoneNumber)=0;
+    virtual void USER_handleStartDial()=0;
+    virtual void USER_handleCallRequest(common::PhoneNumber)=0;
+    virtual void USER_handleCallDrop(common::PhoneNumber) =0;
```

Comment:
Nie możesz się zdecydować na jeden styl stawiania spacji? Dlaczego w 2 jest bez przerw a tutaj z i to jeszcze tak dziko?